front/lib/actions/mcp_internal_actions/servers/agent_router.ts

```diff
@@ -20,12 +20,32 @@ const serverInfo: InternalMCPServerDefinitionType = {
   documentationUrl: null,
 };
 
+export const LIST_ALL_AGENTS_TOOL_NAME = "list_all_published_agents";
+export const SUGGEST_AGENTS_TOOL_NAME = "suggest_agents_for_content";
+
+const SERVER_INSTRUCTIONS = `This toolset provides access to specialized agents available in the workspace.
+A good usage of these tools is:
+- when the user asks for a list of agents or when the user asks for suggestions for an agent.
+- when the agent is not able to handle the user's request and tries to suggest an agent that might be able to handle the request.
+
+# Agent Presentation Format
+Both tools return agents with a "mention" field containing a clickable format (e.g., \`:mention[AgentName]{sId=abc123}\`). This format allows users to directly select and interact with the suggested agents.
```

Comment:
mention markdown directive?

Reply:
I think you need a bit more tokens on what this directive is. You want to be explicit that the directive should be used to display a clickable version of the agent name in the response.